test/unit/argument_parser/format_parse_test.cpp

```diff
@@ -581,6 +581,77 @@ TEST(parse_type_test, parse_error_double_option)
     EXPECT_THROW(parser2.parse(), parser_invalid_argument);
 }
 
+namespace Bar
+{
+enum class Foo
```

Comment:
This is :nail_care: because it's tests, but we don't use CamelCase anywhere else...